src/components/AppSettingsDialog/AppSettingsDialog.vue

```diff
@@ -298,11 +298,12 @@ export default {
 		overflow-y: auto;
 		overflow-x: hidden;
 		padding: 24px;
+		width: 100%;
```

Comment:
This makes sure that only one scrollbar is shown on the right of the modal (would otherwise show a second scrollbar )

Reply:
Couldn't `max-width: 100%` do the trick ?

Reply:
good idea, will try that

Reply:
That doesn't work as it doesn't expand to the width of the element otherwise and would produce an additional scroller...